Modules/Projects/Core/Strings_Misc.json

```
@@ -21,6 +21,12 @@
 				"There's a card here with an odd message on it, about how this area is 'not yet implemented'. Weird. Maybe you should come back some other time."
 			]
 		},
+		{
+			"Key": "SLEEP_NIGHT_CANNOT",
+			"Text": [
+				"It's already night, there's no need to sleep again."
```

Would prefer this mentions the specific option the player chose, so it's clear that sleeping in itself is still okay (since both options will probably be shown side to side):
```suggestion
				"It's already night; there's no need to sleep until night again."
```